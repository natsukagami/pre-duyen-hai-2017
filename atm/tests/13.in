287 224
-8421264271256305020081969112509420639293796519811825248312679948134498409139100651531762281739309323653166330155613494877486150828817379361995853375608435574792874426287227663722521182754542943187636827768961923950902028640534331701293872871870280736935980409478936894447155507204025825
