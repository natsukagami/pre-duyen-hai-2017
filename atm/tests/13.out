999999999999999999999999988935980409478936894447155507204025825
